Sources/Testing/Events/Event.swift

```diff
@@ -252,3 +252,152 @@ extension Event {
     }
   }
 }
+
+// MARK: Snapshot
```

Comment:
Nit: A dash after "MARK: " adds a separator in the Xcode pulldown menu:

```
// MARK: - Snapshot
```

Makes it a bit nicer to navigate in Xcode. :)